src/test/java/com/powsybl/network/conversion/server/NetworkConversionTest.java

```diff
@@ -108,6 +106,149 @@ public void setup() {
         networkConversionService.setReportServerRest(reportServerRest);
     }
 
+    @Test
+    public void testToEquipmentInfos() {
```

Comment:
`NetworkConversionTest` is the class for integration tests.
Move this unit test in `EquipmentInfosServiceTests` class